docs/src/main/paradox/api-reference/instances.md

```diff
@@ -1,314 +1,194 @@
+
+```
+DELETE /v0/data/{orgId}/{domId}/{name}/{version}/{id}/attachment?rev={rev}
+```
 This does not delete the binary attached to an instance, although it prevents from fetching it from the latest review. The binary attached will be still available fetching an older review.
 
-@@@
+#### Example
+
+Request
+:   @@snip [instance-delete.sh](../assets/api-reference/instances/instance-delete-attachment.sh)
+
+Response
+:   @@snip [instance-ref-delete.json](../assets/api-reference/instances/instance-ref-delete-attachment.json)
+
+### Search instances
+
+Follows the general definition of searching in a collection of resources.
+
+```
+GET /v0/data/{orgId}/{domId}/{name}/{version}
+      ?q={full_text_search_query}
+      &filter={filter}
+      &from={from}
+      &size={size}
+      &deprecated={deprecated}
+      &published={published}
+```
+... where 
+
+* `{orgId}` filters the resulting instances to belong to a specific organization.
+* `{domId}` filters the resulting instances to belong to a specific domain.
+* `{name}` filters the resulting instances to have a specific name.
+* `{version}` filters the resulting instances to have a specific version.
+* `{full_text_search_query}` is an arbitrary string that is looked up in the attribute values of the selected instances.
+* `{filter}` is a filtering expression as described in the @ref:[Search and filtering](operating-on-resources.md#search-and-filtering) section.  
+* `{from}` and `{size}` are the listing pagination parameters.  
+* `{deprecated}` selects only instances that have the specified deprecation status.
+
+All query parameters described (`q`, `filter`, `from`, `size`, `deprecated` and `published`) are optional.
 
-### Request path and query parameters
-| Field         | Type          | Description                                                                                                                   |
-| ------------- |-------------  | ---------------------------------------------                                                                                 |
-| orgId         | String        | The unique identifier for the organization                                                                                    | 
-| domainId      | String        | The unique identifier for the domain for this particular organization.                                                        |
-| name          | String        | The unique identifier for the schema for this particular domain.                                                              |
-| version       | String        | The unique identifier for the schema version for this particular schema, following the [semantic format](http://semver.org/)  |
-| rev           | Long]         | The current revision of the instance.                                                                                         |
+The path parameters `/{orgId}/`, `/{domId}/` and `/{name}/` `/{version}/` are optional.
```

Comment:
The path segments (`{orgId}/{domId}/{name}/{version}`) are optional; when used, they constrain the resulting listing to contain only instances that share the same organization, domain ids and schema name and version.  Any of the segments can be omitted but since they are positional it's required that depended segments (to the left) are specified. For example, one can list all the instances within a domain using a simple GET request on `/v0/data/{orgId}/{domId}`.